ledger-core/virtual/integration/method_test.go

```diff
@@ -358,14 +356,20 @@ func TestVirtual_CallMethodAfterPulseChange(t *testing.T) {
 		server.SendMessage(ctx, msg)
 	}
 
+	server.WaitIdleConveyor()
+	server.ResetActiveConveyorFlag()
```

Comment:
No need to reset active conveyor flag, no more

Reply:
fixed